codegen/service.go

```diff
@@ -31,6 +31,7 @@ import (
 
 // ModuleSpec collects the service specifications from thrift file.
 type ModuleSpec struct {
+	CompiledModule *compile.Module `json:"omitempty"`
```

Comment:
It would be helpful to drop a comment here to explain what it is and how it is different from `CompileSpec`.

Do we need the json tag?

Reply:
json tag is used to avoid `null` that marshal-led to golden file